docs/ops/cli.md

```diff
@@ -265,6 +268,9 @@ Action "run" compiles and runs a program.
                                           connect to a different JobManager than
                                           the one specified in the
                                           configuration.
+     -sae,--schutdownOnAttachedExist      If the job mode is not detached and
```

Comment:
Is this supposed to be _exit_ instead of _exist_? How about `--shutdownOnExit`?

Reply:
I also want to keep it as short as possible. The problem is that I think `--shutdownOnExit` might be not informative enough and delude users that shutdown can also happen on exit in detached mode. We have the description for that of course but it needs to be read.